tests/parsers/test.manifestjson.js

```diff
@@ -46,7 +46,7 @@ describe('ManifestJSONParser manifestVersion', function() {
                                                     addonLinter.collector);
     assert.equal(manifestJSONParser.isValid, false);
     var errors = addonLinter.collector.errors;
-    assert.equal(errors[0].code, 'MANIFEST_JSON_INVALID');
+    assert.equal(errors[0].code, 'MANIFEST_FIELD_INVALID');
```

Comment:
You should use `messages.MANIFEST_FIELD_INVALID.code` (the code constant) rather than the string, in case that ever changes or something.